app/controllers/discourse_automation/admin_discourse_automation_scriptables_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module DiscourseAutomation
+  class AdminDiscourseAutomationScriptablesController < ::ApplicationController
+    def index
+      scriptables = DiscourseAutomation::Scriptable
```

Comment:
Can/should that be cached somewhere to avoid re-doing unnecessary work?

Reply:
DiscourseAutomation::Scriptable.all this is already cached